test/150_connect_forget_test.sh

```diff
@@ -15,8 +15,8 @@ assert_peers() {
 
 start_suite "Connecting and forgetting routers after launch"
 
-weave_on $HOST1 launch
-weave_on $HOST2 launch
+weave_on $HOST1 launch-router
+weave_on $HOST2 launch-router
```

Comment:
without the change it's failing for me with
````
test #6 "exec_on 192.168.48.11 c1 ping -nq -W 1 -c 1 10.2.1.7" failed:
	program terminated with code 1 instead of 0
````
But why? Please explain.